Created state for filters

pigos4 opened the pull request.

pigos4 committed:
Created state for filters

pigos4 committed:
Merge branch 'main' of https://github.com/CodeYourFuture/ldn8-horizon-chatterbox-talent into State-forFilters

pigos4 committed:
Reset tick boxes

RitaGlushkova reviewed (approved):
Just need to tidy up a little. Well done!

pigos4 committed:
Clean the function and format all files

RitaGlushkova reviewed (changes requested):
@pigos4  I tested the filters state functionality and it seems to be working one click late.
For example, if I add `onSite, Hybrid and Remote` options it shows `onSite, Hybrid` and only when I click on the next option (let's say `HR`), then it shows `onSite, Hybrid and Remote` but no `HR`. So it is always one click late.

RitaGlushkova committed:
change state-forFilters "split checkbox filters & changed keys"

I made a few changes for Daniel's PR.
1. I split CheckBoxFilters into 5 different components because we will have to apply very different styles to them.
2. I changes key names into state "filters" to match exactly how it is in original data but displays names on Modal human friendly.

RitaGlushkova committed:
style for search and filter btns

RitaGlushkova committed:
Update index.tsx

pigos4 committed:
Merge pull request #39 from CodeYourFuture/Rita-b-State-forFilters-btns

craig-dsilva reviewed (approved):
Good job adding the prettier config
